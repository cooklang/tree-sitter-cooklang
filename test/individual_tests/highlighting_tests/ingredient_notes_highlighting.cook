@flour{2%cups}(sifted and leveled)