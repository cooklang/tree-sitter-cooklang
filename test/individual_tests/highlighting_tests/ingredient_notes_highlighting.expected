(recipe
  (step
    (ingredient
      name: (ingredient_text)
      (amount
        (quantity)
        (units))
      note: (note_text)))